### DISCUSSION TITLE
On the art of visualizations

### DISCUSSION THREAD

**Discussion by caspervanengelenburg** [udlbook/udlbook#37] [Q&A]
Dear Simon, what software did you use to create the (very consistent in appearance) figures in your written work? (It seems a combination of plotting in seaborn, whenever real information was needed, and Illustrator, is that right?)

**Comment by udlbook** [comment]
I actually make them in Inkscape, with the textext plugin.  All of the text and equations in the figures are made with this plugin.  When I reproduce other people's data, I extract the page from the pdf, import it into Inkscape, and recolor.  I make all the axis labels and ticks by hand.
